code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -440,6 +442,9 @@
 	linked_artillery = null
 	to_chat(src, span_notice("NOTICE: Connection closed with linked mortar."))
 
+/mob/living/silicon/ai/proc/set_random_ai_screen()
+	var/list/iconstates = GLOB.ai_core_display_screens
+	icon_state = resolve_ai_icon(pick(iconstates))
```

Comment:
why is this a separate proc? Its only used once.

Reply:
I figured it could be used a bunch of times in the future.

Reply:
We tend to want to avoid 'just in case' code. Unless you have specified use cases in mind you'd be using for a future pr after this is merged, its better to keep it inlined. If someone wants the functionality in the future, its like 5 seconds to make it into a proc.

Reply:
Ok I'll fix